frontend/packages/knative-plugin/src/components/traffic-splitting/TrafficModalRevisionsDropdownField.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { set, cloneDeep } from 'lodash';
+import { useField } from 'formik';
+import { DropdownField } from '@console/shared';
+
+type TrafficModalRevisionsDropdownFieldProps = {
+  revisionItems: any;
```

Comment:
do we know type for it

Reply:
It is an object of type {string(revisionName): string(revisionName)}
Should I define a type for it in utils to use?

Reply:
There is already a type `RevisionItems` in utils, got overlooked
updated it with type